documentation/boltspec_reference.md

```diff
@@ -4,6 +4,180 @@ The `BoltSpec` helper library includes several functions to help write unit
 tests for plans. For more information about writing unit tests for plans, and
 how to use these functions, see [Testing plans](testing_plans.md).
 
+## Any stubs
+
+Plans often execute sub-plans with the `run_plan` function to build complex
+workflows. When testing these plans, it might be helpful to execute any
+sub-plans as well without needing to stub or mock the plan. To support this,
+`BoltSpec` offers two different modes:
+
+### `allow_any_plan`
+
+The `allow_any_plan` function stubs the [`run_plan`
```

Comment:
```suggestion
The `allow_any_plan` function stubs all invocations of the [`run_plan`
```

We should clarify that it stubs _all_ invocations of the plan function. Same for all of the other `allow_any_*` functions.